pkg/collector/corechecks/ebpf/probe/oom_kill.go

```diff
@@ -41,12 +45,70 @@ type OOMKillProbe struct {
 }
 
 func NewOOMKillProbe(cfg *ebpf.Config) (*OOMKillProbe, error) {
-	compiledOutput, err := runtime.OomKill.Compile(cfg, []string{"-g"}, statsd.Client)
+	if cfg.EnableCORE {
+		probe, err := loadCOREProbe(cfg)
+		if err == nil {
+			return probe, err
+		}
+
+		if !cfg.AllowRuntimeCompiledFallback {
+			return nil, fmt.Errorf("error loading CO-RE oom-kill probe: %s. set system_probe_config.allow_runtime_compiled_fallback to true to allow fallback to runtime compilation.", err)
+		}
+		log.Warnf("error loading CO-RE oom-kill probe: %s", err)
```

Comment:
can the message explains that it will default to legacy probe compilation?

Reply:
Definitely!